app/models/ingredient.rb

```diff
@@ -1,3 +1,9 @@
 class Ingredient < ApplicationRecord
+   has_many :recipe_ingredients
+   has_many :recipes, through: :recipe_ingredients
    validates_presence_of :name, :cost
+
+   def self.sort_by_name
+      order(:name)
```

Comment:
This logic is encapsulated nicely. For future ICs, just make sure you test these model methods in a model spec as well.